src/Model/SessionManager.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Model;
+
+class SessionManager extends AbstractManager
```

Comment:
Good idea to have a SessionManager class to deal with session problems.
But, it's not a *Manager like others that are supposed to be used to interact with persistant data.
So this class should not extends AbstractManager
And for more readability, it should probably have another name... SessionHandler ?